cmd/soroban-rpc/internal/test/metrics_test.go

```diff
@@ -0,0 +1,22 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMetrics(t *testing.T) {
+	test := NewTest(t)
+	metrics, err := test.daemon.PrometheusRegistry().Gather()
+	assert.NoError(t, err)
```

Comment:
I think that *require* here would work better, since the rest of the test could easily stop here.